README.md

```diff
@@ -203,7 +209,44 @@ To get the webhook:
 ```powershell
 az webapp deployment container config -n jackiemaw-do-me -g CreditSuisse21_JacquelineUngerer_ProjectExercise -e true
 ```
-  
+### Trouble-shooting
+
+View application logs here:
+        https://jackiemaw-do-me.scm.azurewebsites.net/api/logstream
+
+## Deploying with Kubernetes
+
+You can deploy the application manually to Kubernetes:
+
+```powershell
+minikube start
+
+kubectl create secret generic secrets --from-literal=db_connectionstring='<mongodb connection string>'   --from-literal=client_secret='<github client secret>' --from-literal=flask_secret='<flask session secret>'
+
+kubectl apply -f deployment.yaml
+kubectl apply -f service.yaml
+kubectl port-forward service/module-14 7080:80 4
+```
+
```

Comment:
These instructions are correct given that you removed `imagePullPolicy: Never` and you used the full Docker Hub repository name. 

But if experimenting locally, you'd probably want to use a local image, so you'd need to build it and use `minikube image load`

Reply:
You might also want to point out that the reader can access it via port 7080 so they don't need to read/understand the kubectl commands.